gitbook/result/tryCreate.md

```diff
@@ -37,15 +37,19 @@ type Longitude = private Longitude of float with
 Let's assume that we have few more similar types as below
 
 ```fsharp
-type Longitude = private Longitude of float with
+type Latitude = private Latitude of float with
   member this.Value =
-    let (Longitude lng) = this
-    lng
-  static member TryCreate (lng : float) =
-    if lng > -180. && lng < 180. then
-      Ok (Longitude lng)
+    let (Latitude lat) = this
+    lat
+
+  // float -> Result<Latitude, string>
+  static member TryCreate (lat : float) =
+    if lat >= -90. && lat <= 90. then
+      Ok (Latitude lat)
     else
-      sprintf "%A is a invalid longitude value" lng |> Error
+      sprintf "%A is a invalid latitude value" lat |> Error
```

Comment:
```suggestion
      Error $"%A{lat} is a invalid latitude value"
```
(patens on prev line and pipe this line is jarring - pick one I say ;)

Reply:
Same for the `Longitude` type.